pkg/migrations/op_add_column.go

```diff
@@ -99,9 +99,17 @@ func (o *OpAddColumn) Start(ctx context.Context, conn db.DB, latestSchema string
 		tableToBackfill = table
 	}
 
-	table.AddColumn(o.Column.Name, &schema.Column{
-		Name: TemporaryName(o.Column.Name),
-	})
+	tmpColumn := &schema.Column{
```

Comment:
to avoid mistakes if a new field is added to `schema.Column` we probably want to clone `o.Column` then change the name maybe?

Reply:
Ok, I am refactoring the code.

Reply:
These are two different `Column` types, I cannot really clone them. I added a function we could call to translate from `migrations.Column` to `schema.Column`.